File: src/main/java/xyz/hiziki/util/Prefix.java
package xyz.hiziki.util;

import org.bukkit.ChatColor;
import org.bukkit.entity.Player;

public class Prefix
{
    public Prefix(Player p, String msg)
    {
        p.sendMessage(ChatColor.AQUA + "[" + ChatColor.GREEN + "SimpleSetHome" + ChatColor.AQUA + "]" + msg);
    }
}

File: src/main/java/xyz/hiziki/util/Prefix.kt
package xyz.hiziki.util

import org.bukkit.ChatColor
import org.bukkit.entity.Player

class Prefix(p : Player, msg : String)
{
    init
    {
        p.sendMessage(ChatColor.AQUA.toString() + "[" + ChatColor.GREEN + "SimpleSetHome" + ChatColor.AQUA + "]" + msg)
    }
}